### ISSUE TITLE
Could you provide the logfile of pretraining process?

### ISSUE THREAD

**Issue by forjiuzhou** [jackroos/VL-BERT#18]
Hi, Thanks for the great repo. 

I want to get an understanding of what happened during the pretraining, especially the train and validation loss change. 

In other work like lxmert and my own experiment, The MLM loss is highly overfitted. So I wander is that the same thing of VL-BERT? Considering the specia mask operation of VL-BERT.

Is it possible that you can share the logfile of the pretraining process?

**Comment by jackroos** [comment]
@forjiuzhou Sorry for the late reply. The training accuracy of VL-BERT Large is listed as below:
* Masked Language Modeling with Visual Clues: 67.0%
* Masked Language Modeling on Text-only Corpus: 65.2%
* Masked RoI Classification with Linguistic Clues: 70.1%
And I think MLM accuracy is reasonable, not highly overfitted in VL-BERT. Hope it would help you, Thanks!

**Comment by liming0791** [comment]
> @forjiuzhou Sorry for the late reply. The training accuracy of VL-BERT Large is listed as below:
> 
> * Masked Language Modeling with Visual Clues: 67.0%
> * Masked Language Modeling on Text-only Corpus: 65.2%
> * Masked RoI Classification with Linguistic Clues: 70.1%
> 
> And I think MLM accuracy is reasonable, not highly overfitted in VL-BERT. Hope it would help you, Thanks!

Thanks for you information !
And what about the training accuracy of VL-BERT Base, under precomputed features configure and end-to-end configure ?
